System.Compression.Primitives/System/IO/Compression/Brotli/BrotliStream.cs

```diff
@@ -34,33 +32,8 @@ public partial class BrotliStream : Stream
         private int _readOffset = 0;
         Decoder _decoder;
         Encoder _encoder;
-       /* public BrotliStream(Stream baseStream, CompressionMode mode, bool leaveOpen, int BuffSize, uint windowSize, uint quality)
-        {
-            if (baseStream == null) throw new ArgumentNullException("baseStream");
-                       _mode = mode;
-            _stream = baseStream;
-            LeaveOpen = leaveOpen;
-            if (_mode == CompressionMode.Compress)
-            {
-                _encoder.InitializeEncoder();
-                _encoder.SetQuality(quality);
-                _encoder.SetWindow(windowSize);
-            }
-            else
-            {
-                _decoder.InitializeDecoder();
-            }
-            BufferSize = BuffSize;
-            BufferIn = Marshal.AllocHGlobal(BufferSize);
-            BufferOut = Marshal.AllocHGlobal(BufferSize);
-            NextIn = BufferIn;
-            NextOut = BufferOut;
-            AvailOut = new IntPtr((uint)BuffSize);
-            //BrotliNative.BrotliEncoderSetCustomDictionary(State,10, Dict);
-        }*/
-
         public BrotliStream(Stream baseStream, CompressionMode mode, bool leave_open, int BuffSize, uint windowsize, uint quality) : this(baseStream, mode, leave_open, BuffSize) {
-            if (_mode == CompressionMode.Decompress) throw new Exception(); // quality and windowsize is ambitious for Decompress mode
+            if (_mode == CompressionMode.Decompress) throw new System.IO.IOException("quality and windowsize is ambitious for Decompress mode");
```

Comment:
It means that it is no need and sense to give decompressor quality and windowsize